--- kdbg-2.0.0/po/Makefile.in.orig	2005-07-14 21:27:12.000000000 +0200
+++ kdbg-2.0.0/po/Makefile.in	2005-07-31 12:31:24.151705760 +0200
@@ -283,8 +283,8 @@
 xdg_menudir = @xdg_menudir@
 #>- POFILES = AUTO
 #>+ 2
-POFILES = zh_CN.po da.po de.po cs.po es.po fr.po hr.po hu.po ja.po it.po nb.po nn.po pl.po pt.po ro.po ru.po sk.po sr.po sv.po tr.po
-GMOFILES = zh_CN.gmo da.gmo de.gmo cs.gmo es.gmo fr.gmo hr.gmo hu.gmo ja.gmo it.gmo nb.gmo nn.gmo pl.gmo pt.gmo ro.gmo ru.gmo sk.gmo sr.gmo sv.gmo tr.gmo
+POFILES = zh_CN.po da.po de.po cs.po es.po fr.po hr.po hu.po ja.po it.po nb.po nn.po pl.po pt.po ro.po ru.po sk.po sr@Latn.po sv.po tr.po
+GMOFILES = zh_CN.gmo da.gmo de.gmo cs.gmo es.gmo fr.gmo hr.gmo hu.gmo ja.gmo it.gmo nb.gmo nn.gmo pl.gmo pt.gmo ro.gmo ru.gmo sk.gmo sr@Latn.gmo sv.gmo tr.gmo
 #>- all: all-am
 #>+ 1
 all: all-nls docs-am  all-am
@@ -529,9 +529,9 @@
 sk.gmo: sk.po
 	rm -f sk.gmo; $(GMSGFMT) -o sk.gmo $(srcdir)/sk.po
 	test ! -f sk.gmo || touch sk.gmo
-sr.gmo: sr.po
-	rm -f sr.gmo; $(GMSGFMT) -o sr.gmo $(srcdir)/sr.po
-	test ! -f sr.gmo || touch sr.gmo
+sr@Latn.gmo: sr@Latn.po
+	rm -f sr@Latn.gmo; $(GMSGFMT) -o sr@Latn.gmo $(srcdir)/sr@Latn.po
+	test ! -f sr@Latn.gmo || touch sr@Latn.gmo
 sv.gmo: sv.po
 	rm -f sv.gmo; $(GMSGFMT) -o sv.gmo $(srcdir)/sv.po
 	test ! -f sv.gmo || touch sv.gmo
@@ -541,7 +541,7 @@
 
 #>+ 3
 clean-nls:
-	-rm -f  zh_CN.gmo da.gmo de.gmo cs.gmo es.gmo fr.gmo hr.gmo hu.gmo ja.gmo it.gmo nb.gmo nn.gmo pl.gmo pt.gmo ro.gmo ru.gmo sk.gmo sr.gmo sv.gmo tr.gmo
+	-rm -f  zh_CN.gmo da.gmo de.gmo cs.gmo es.gmo fr.gmo hr.gmo hu.gmo ja.gmo it.gmo nb.gmo nn.gmo pl.gmo pt.gmo ro.gmo ru.gmo sk.gmo sr@Latn.gmo sv.gmo tr.gmo
 
 #>+ 10
 install-nls:
