--- kdbg-1.9.7/po/Makefile.in.orig	2005-05-08 14:09:51.948368992 +0200
+++ kdbg-1.9.7/po/Makefile.in	2005-05-08 14:12:01.745636808 +0200
@@ -283,8 +283,8 @@
 xdg_menudir = @xdg_menudir@
 #>- POFILES = AUTO
 #>+ 2
-POFILES = da.po de.po cs.po es.po fr.po hr.po hu.po ja.po it.po nb.po nn.po pl.po pt.po ro.po ru.po sk.po sr.po sv.po tr.po zh_CN.GB2312.po
-GMOFILES = da.gmo de.gmo cs.gmo es.gmo fr.gmo hr.gmo hu.gmo ja.gmo it.gmo nb.gmo nn.gmo pl.gmo pt.gmo ro.gmo ru.gmo sk.gmo sr.gmo sv.gmo tr.gmo zh_CN.GB2312.gmo
+POFILES = da.po de.po cs.po es.po fr.po hr.po hu.po ja.po it.po nb.po nn.po pl.po pt.po ro.po ru.po sk.po sr@Latn.po sv.po tr.po zh_CN.GB2312.po
+GMOFILES = da.gmo de.gmo cs.gmo es.gmo fr.gmo hr.gmo hu.gmo ja.gmo it.gmo nb.gmo nn.gmo pl.gmo pt.gmo ro.gmo ru.gmo sk.gmo sr@Latn.gmo sv.gmo tr.gmo zh_CN.GB2312.gmo
 #>- all: all-am
 #>+ 1
 all: all-nls docs-am  all-am
@@ -478,7 +478,7 @@
 
 
 #>+ 2
-KDE_DIST=tr.gmo sv.gmo sr.gmo nn.po da.po ru.gmo nb.gmo de.po kdbg.pot ja.gmo nb.po fr.po cs.po tr.po sk.gmo ro.po ja.po it.gmo de.gmo es.po pt.gmo es.gmo it.po da.gmo zh_CN.GB2312.po zh_CN.GB2312.gmo sr.po pl.gmo pt.po hr.po ru.po fr.gmo ro.gmo cs.gmo nn.gmo Makefile.in hr.gmo sv.po hu.po pl.po sk.po hu.gmo Makefile.am 
+KDE_DIST=tr.gmo sv.gmo sr@Latn.gmo nn.po da.po ru.gmo nb.gmo de.po kdbg.pot ja.gmo nb.po fr.po cs.po tr.po sk.gmo ro.po ja.po it.gmo de.gmo es.po pt.gmo es.gmo it.po da.gmo zh_CN.GB2312.po zh_CN.GB2312.gmo sr@Latn.po pl.gmo pt.po hr.po ru.po fr.gmo ro.gmo cs.gmo nn.gmo Makefile.in hr.gmo sv.po hu.po pl.po sk.po hu.gmo Makefile.am 
 
 #>+ 61
 da.gmo: da.po
@@ -529,9 +529,9 @@
 sk.gmo: sk.po
 	rm -f sk.gmo; $(GMSGFMT) -o sk.gmo $(srcdir)/sk.po
 	test ! -f sk.gmo || touch sk.gmo
-sr.gmo: sr.po
-	rm -f sr.gmo; $(GMSGFMT) -o sr.gmo $(srcdir)/sr.po
-	test ! -f sr.gmo || touch sr.gmo
+sr@Latn.gmo: sr@Latn.po
+	rm -f sr@Latn.gmo; $(GMSGFMT) -o sr@Latn.gmo $(srcdir)/sr@Latn.po
+	test ! -f sr@Latn.gmo || touch sr@Latn.gmo
 sv.gmo: sv.po
 	rm -f sv.gmo; $(GMSGFMT) -o sv.gmo $(srcdir)/sv.po
 	test ! -f sv.gmo || touch sv.gmo
@@ -544,7 +544,7 @@
 
 #>+ 3
 clean-nls:
-	-rm -f  da.gmo de.gmo cs.gmo es.gmo fr.gmo hr.gmo hu.gmo ja.gmo it.gmo nb.gmo nn.gmo pl.gmo pt.gmo ro.gmo ru.gmo sk.gmo sr.gmo sv.gmo tr.gmo zh_CN.GB2312.gmo
+	-rm -f  da.gmo de.gmo cs.gmo es.gmo fr.gmo hr.gmo hu.gmo ja.gmo it.gmo nb.gmo nn.gmo pl.gmo pt.gmo ro.gmo ru.gmo sk.gmo sr@Latn.gmo sv.gmo tr.gmo zh_CN.GB2312.gmo
 
 #>+ 10
 install-nls:
